test/minimal.cpp

```diff
@@ -0,0 +1,6 @@
+#define JC_TEST_USE_DEFAULT_MAIN
+#include <jc_test.h>
+
+TEST_CASE(multiply) {
+  ASSERT_EQ(4, 2*2);
+}
```

Comment:
Great!
Please add it to the `compile.sh`/`compile_cl.bat` so that it is also actually compiled.
And also `run_tests-bat`so that it's also run.
(yes, adding new test files can be a bit cleaner imho, I'll get to that at some point)

Reply:
Added.